domain/src/main/java/com/kdjj/domain/repository/RecipeImageRepository.kt

```diff
@@ -15,4 +15,6 @@ interface RecipeImageRepository {
         uri: String,
         fileName: String
     ): Result<String>
+
+    fun isUriExists(uri: String): Boolean
```

Comment:
같은 파일에서 컨벤션이 다른거 같아요!
변경부탁드립니다!
```suggestion
    fun isUriExists(
        uri: String
    ): Boolean
```